music.css

```diff
@@ -539,7 +539,6 @@ html:not(.dim) #player.fix {
 #volume:before { content: "\f211" }
 #volume.muted:before { content: "\f210" }
 
-.clear:before { content: "\f217" }

```

Comment:
There are still two buttons using the `.clear` class (the Clear button in the playlist and the one next to the filter field).